### ISSUE TITLE
20221026 ci is running on master after merge?

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#5011]
https://github.com/radiasoft/sirepo/actions/workflows/python-ci.yml shows CI is running on master. @rorour would you look at asap? This slows down releases, because the tests run twice. Maybe I'm missing something, and we expect it to run on master. If so, please explain.

**Comment by rorour** [comment]
Yes, we expect it to run on master. I think [this comment/issue](https://github.com/radiasoft/pykern/issues/223#issuecomment-1175450151) addressed that, [see also this](https://github.com/radiasoft/pykern/pull/246#issue-1302552964)

> In order to get the commit sha matching HEAD from the API, I changed the post-merge action to run on master/main instead of the merged branch.

I originally had tests running on the feature branch for the hypothetical merge to master, but because of pykern ci-check, it needs to actually run on master.

I'm happy to talk about it further if that doesn't clear it up.
